spec/koans.js

```diff
@@ -8,31 +8,31 @@ describe('`let` restricts the scope of the variable to the current block - ', ()
   describe('`let` vs. `var`.', () => {
     it('`var` works as usual, it does not restricts scope', () => {
       if (true) {
-        /*You should add your code in here*/
+       varX = true;
       }
-      // expect(varX).toBe(true);
+      expect(varX).toBe(true);
     });
 
     it('`let` restricts scope to inside the block', () => {
-      /*var or const? letX = false*/
+      letX= false/*var or const? letX = false*/
       if (true) {
-        /*var or const? letX = true*/
+       constX= true /*var or const? letX = true*/
       }
-      //expect(letX).toBe(false);
+      expect(letX).toBe(false);
     });
 
     it('`var` does not restricts scope to inside the block in `for` loops', () => {
-      /*var or let? counter = 100*/
-      /*for (var or let? counter = 1; counter < 50; counter++){}*/
+      //let counter = 100 /*var or let? counter = 100*/
+     for (var counter = 1; counter < 50; counter++){};
 
-      //expect(counter).toBe(50);
+      expect(counter).toBe(50);
```

Comment:
here; you are supposed to change both the variable declarations, you can put var twice, because the same variable will just be reassigned, and since var is not block scoped, the value of this variable after the for loop will still keep the memory of its reassignment inside the loop (it would not have worked with let)
```js
    
    it('`var` does not restricts scope to inside the block in `for` loops', () => {
      var counter = 100;
      for (var counter = 1; counter < 50; counter++){}

      expect(counter).toBe(50);
    });
```